Package.swift

```diff
@@ -1,12 +1,16 @@
-// swift-tools-version:3.1 – swiftlint:disable file_header
-// The swift-tools-version declares the minimum version of Swift required to build this package.
-
+// swift-tools-version:4.2
 import PackageDescription
 
 let package = Package(
     name: "AnyMenu",
-    dependencies: [
-//        .package(url: "https://github.com/Flinesoft/HandySwift.git", .upToNextMajor(from: "2.3.0")),
-//        .package(url: "https://github.com/Flinesoft/HandyUIKit.git", .upToNextMajor(from: "1.4.0")),
+    // platforms: [.iOS("8.0"), .macOS("10.10"), .tvOS("9.0"), .watchOS("2.0")],
```

Comment:
Not needed, but can stay. Especially given that AnyMenu doesn't support macOS, tvOS & watchOS ;)